src/Compiler/CompilerOptions.hpp

```diff
@@ -56,6 +56,10 @@ void setTargetArch(const std::string &arch);
 std::string getTargetArchOption();
 void setTargetCPU(const std::string &cpu);
 std::string getTargetCPUOption();
+// SetTargetAccel add given accleator to the list, reset when str="RESET".
```

Comment:
accleator -> accelerator

Reply:
Also document the function return semantics (suggest to use a bool rather than an int for the return type).

Reply:
Return an int because the function that uses it in the interface follows the "0" fine, "nonzero" is error code.
Now if you want it, I can return a boolean here and then convert the true/false to an zero/nonzero in that interface.

Reply:
Let me know your preference, and will add comments either way.

Reply:
I am fine with following the zero/non-zero interface.